build.sh

```diff
@@ -0,0 +1,161 @@
+     exit 1
+  fi
+}
+
+################################################################################
+function printInfo {
+################################################################################
+echo ---------------------------------------------------------------------------
+echo $*
+echo ---------------------------------------------------------------------------
+}
+
+################################################################################
+# END DECLARATION FUNCTIONS / START OF SCRIPT
+################################################################################
+
+################################################################################
+# Test for commands used in this script
+################################################################################
+testForCommands="dirname git date java javac"
+
+for command in $testForCommands
+do 
+  type $command >> /dev/null
+  if [ $? -ne 0 ]; then
+    echo "Error: command '$command' is not installed!"
+    exit 1
+  fi
+done
+
+################################################################################
+# Determine directory of script. 
+################################################################################
+ACTUAL_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+################################################################################
+# Check parameters
+################################################################################
+checkParameters $*
+
+################################################################################
+# Determine repo name 
+################################################################################
+REPO_URL=`git remote get-url origin`
+REPO_NAME=${REPO_URL##*/}
+
+printInfo Build microservice of $REPO_NAME at $INSTALLATION_DIRECTORY
+
+
+################################################################################
+# Build service
+################################################################################
+
+echo Build service...
+./gradlew -Prelease clean build
+
+
+echo Copy configuration to $INSTALLATION_DIRECTORY...
+find . -name application-default.properties -exec cp '{}' $INSTALLATION_DIRECTORY/application.properties \;
+
+echo Copy jar file to $INSTALLATION_DIRECTORY...
+find . -name "$REPO_NAME*.jar" -exec cp '{}' $INSTALLATION_DIRECTORY \;
+
+echo Create config directory
+mkdir $INSTALLATION_DIRECTORY/config
+
+echo Create lib directory
+mkdir $INSTALLATION_DIRECTORY/lib
+
+###############################################################################
+# Create run script
+################################################################################
+printInfo Create run script ...
+
+cd $INSTALLATION_DIRECTORY
+
+# Determine name of jar file.
+jarFile=(`ls $REPO_NAME*.jar`)
+
+echo "#!/bin/bash"                                                                              >  $INSTALLATION_DIRECTORY/run.sh
```

Comment:
So, when building locally on OSX using `bash build.sh installdir` I do get those errors, and I do not have a fix for that as I do not fully understand the problem yet:

```
---------------------------------------------------------------------------
Create run script ...
---------------------------------------------------------------------------
build.sh: line 143: installme/run.sh: No such file or directory
build.sh: line 144: installme/run.sh: No such file or directory
build.sh: line 145: installme/run.sh: No such file or directory
build.sh: line 146: installme/run.sh: No such file or directory
build.sh: line 147: installme/run.sh: No such file or directory
build.sh: line 148: installme/run.sh: No such file or directory
build.sh: line 149: installme/run.sh: No such file or directory
build.sh: line 150: installme/run.sh: No such file or directory
build.sh: line 151: installme/run.sh: No such file or directory
build.sh: line 152: installme/run.sh: No such file or directory
build.sh: line 153: installme/run.sh: No such file or directory
build.sh: line 154: installme/run.sh: No such file or directory
build.sh: line 155: installme/run.sh: No such file or directory
build.sh: line 156: installme/run.sh: No such file or directory
chmod: installme/run.sh: No such file or directory
.
---------------------------------------------------------------------------
Now you can start the service by calling installme/run.sh
---------------------------------------------------------------------------
```

Reply:
Ahrg. I always tried with absolute paths which works fine.